src/Components/Components/src/RenderTree/RenderTreeBuilder.cs

```diff
@@ -516,6 +516,24 @@ public void AddMultipleAttributes<T>(int sequence, IEnumerable<KeyValuePair<stri
             }
         }
 
+        /// <summary>
+        /// <para>
+        /// Indicates that the preceding attribute represents an event handler
+        /// whose execution updates the attribute with name <paramref name="updatesAttributeName"/>.
+        /// </para>
+        /// <para>
+        /// This information is used by the rendering system to determine whether
+        /// to accept a value update for the other attribute when receiving a
+        /// call to the event handler.
```

Comment:
@SteveSandersonMS - are you concerned that this won't really be possible to do with *manual* two-way binding?

Reply:
Yes, and I'd be happy to discuss this whole design with you next week.

My thought was that later on, we'd also add another parameter to event handlers, e.g., `@oninput:updateField="value"` so that it is possible to do this with a manual two-way binding. It would be optional, since it's up to you whether or not the binding enforces consistency synchronously when the event is processed.

I also thought it would be acceptable for this to be a `@bind`-only feature for preview 7, and maybe longer, since adding `@onevent:updateField` is not urgent.